features/steps/pets_steps.py

```diff
@@ -0,0 +1,55 @@
+######################################################################
```

Comment:
Bringing in this file for now, we'll make edits to it later. Not sure if Github will let us rename the file later on